src/scss/sections/_product-carousel.scss

```diff
@@ -0,0 +1,138 @@
+.product-carousel {
+  .shop-all {
+    min-width: calc-em(154);
+    @include media-breakpoint-up(lg) {
+      min-width: calc-em(216);
+    }
+  }
+
+  .carousel--centered {
+    padding-top: map-get($spacers, 2);
+  }
+}
+
+.product-card {
+  min-height: calc-em(533);
+
+  &__title {
+    font-size: calc-em(22);
+    padding: 0;
+    margin-top: auto;
+    min-height: calc-em(54);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__price {
+    margin-top: auto;
+
+    .price {
+      font-size: calc-em(22);
+    }
+  }
+
+  &__badge {
+    position: absolute;
+    width: calc-em(66);
+    height: calc-em(66);
+    right: calc-em(-5);
+    top: calc-em(-20);
+
+    @include media-breakpoint-up(lg) {
+      width: calc-em(90);
+      height: calc-em(90);
+    }
+  }
+
+  .review-stars {
+    margin-right: calc-em(5);
+    line-height: calc-em(15);
+
+    i {
+      @include font-size($font-size-base);
+
+      @include media-breakpoint-up(lg) {
+        @include font-size($h3-font-size);
+      }
+    }
+  }
+
+  .shop-overlay {
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    display: none;
+    bottom: 0;
+    left: 0;
+    padding-bottom: calc-em(58);
+    background-image: linear-gradient(180deg, rgba(255, 255, 255, 0.9) 0%, white 100%);
+    visibility: visible;
+    transition: all 300ms ease-in-out;
+
+    &:hover {
+      display: flex;
+    }
+  }
+
+  .btn-choose:hover {
+    + .shop-overlay  {
+      display: flex;
+    }
+  }
+
+  .btn {
+    max-width: calc-em(165);
+    @include media-breakpoint-up(lg) {
+      max-width: 100%;
+    }
+  }
+}
+
+.item-swatch {
+  .item {
```

Comment:
use BEM convention, this element name should be variant-swatch and the children variant-swatch__item